set.mm

```diff
@@ -189193,7 +189241,7 @@ implemented as a function (a set of ordered pairs)".  The current
        sets: without this, ~ ipsstr and theorems like it will have many sethood
        assumptions, and may not even be usable in the empty context.  Instead,
        the sethood assumption is deferred until it is actually needed, e.g.,
-       ~ ipsbase , which requires that the base set is a set but not any of the
+       ~ ipsbase , which requires that the base set be a set but not any of the
```

Comment:
?

Reply:
That was the `minor comment edits: subjunctive` of the commit message. I do not know German, but in English, see Paragraph 4 of https://www.grammar.com/situations-requiring-the-subjunctive-mood/

Reply:
My native speaker intuition doesn't have a big problem with either one but I guess prefers `is`.

If I'm correctly reading that article it is saying to use `is` here, as `be` is archaic:

```
In today’s writing, we [rarely](https://www.definitions.net/definition/rarely) see the use of the main verb be to show the [subjunctive](https://www.definitions.net/definition/subjunctive) mood of am, is, are. [These](https://www.definitions.net/definition/These) uses are [archaic](https://www.definitions.net/definition/archaic) or form [traditional](https://www.definitions.net/definition/traditional) idioms:

    be that as it may …. If God be for us, who can be [against](https://www.definitions.net/definition/against) us? far be it from me ….
```

Reply:
I'd preferred the old version, but as always, my English is not good enough to decide anything.

Reply:
@jkingdon : you're quoting Paragraph 2 ("Situations contrary to fact") whereas I mentioned Paragraph 4 ("Suggestion and necessity"). In Paragraph 4, the example is
> This emergency requires that they be there.

(Note that in Paragraph 2, the preferred form is "were", e.g., "If I were you", "If god were for us", and not "am"/"is".)

Given that set.mm is a written reference work, I think it should't be too colloquial.  But if you think I'm really nitpicking then I'll change it back.  (I tried to make that correction in set.mm silently because I'm not a grammar nazi in the sense that I do not correct people for making similar kinds of mistakes in French, written or spoken; I try to keep being liberal in what I accept and conservative in what I produce!)

Reply:
I'm fine with going with your preference here. As far as I can tell either one is grammatical.